packages/esm-patient-registration-app/src/patient-registration/field/dob/dob.component.tsx

```diff
@@ -25,7 +25,8 @@ export const DobField: React.FC = () => {
   const { t } = useTranslation();
   const {
     fieldConfigurations: { dateOfBirth },
-  } = useConfig() as RegistrationConfig;
+  } = useConfig<RegistrationConfig>();
+  const allowEstimatedBirthDate = dateOfBirth?.useEstimatedDateOfBirth?.enabled;
```

Comment:
```suggestion
  const allowEstimatedBirthDate = dateOfBirth?.useEstimatedDateOfBirth?.enabled ?? false;
```

Reply:
We should also have a check in here for if the patient's birthdate is already estimated, e.g., 

```suggestion
  const allowEstimatedBirthDate = (dateOfBirth?.useEstimatedDateOfBirth?.enabled ?? false) || (Boolean(birthdate) && dobknown);
```

Which requires moving this a bit.

Reply:
Added the || condition in the render